tests/e2e/tests/misc/minimal-config.ts

```diff
@@ -10,5 +10,15 @@ export default function () {
         main: 'main.ts'
       }]
     })))
+    .then(() => ng('build'))
+    .then(() => writeMultipleFiles({
+      './src/script.js': 'console.log(\'hello\')',
+      'angular-cli.json': JSON.stringify({
+        apps: [{
+          root: 'src',
+          scripts: ['./script.js']
+        }]
+      }),
+    }))
     .then(() => ng('build'));
```

Comment:
Could you verify that `console.log('hello')` is in the output? Alternatively, does `ng serve` still work? I'd rather test that.